static/styles/home.css

```diff
@@ -67,4 +67,22 @@ img {
 
 .btn-filter {
   max-width: 150px;
+}
+
+.navigation{
+  margin: auto;
+  display: flex;
+}
+
+.navigation li a{
+  color: white;
+  background-color: var(--bs-indigo);
+}
+
+.pagination{
+  margin: auto;
+}
+.navigation li a:hover{
+  color: black;
```

Comment:
I would change the color of the text when hovering to a slightly lighter color, #ffc107 for example